### DISCUSSION TITLE
Support for :is and :where selectors inside style.

### DISCUSSION THREAD

**Discussion by minht11** [vanilla-extract-css/vanilla-extract#201] [Ideas]
Could support be added for [`:is` selector](https://developer.mozilla.org/en-US/docs/Web/CSS/:is), so use case like this would work:

```js
style({
  background: 'red',
  selectors: {
    '&:is(:focus-within, :hover)': {
      background: 'blue',
    }
  }
})
```
There probably would be some restrictions to it, but since `:is` selector references element itself it shouldn't be too much of a problem.
You can use `globalStyle` as an alternative right now, but it doesn't feel as clean.

**Comment by mattcompiles** [comment]
This was resolved in #204
